extensions/windows-event-log/ConsumeWindowsEventLog.cpp

```diff
@@ -447,7 +447,7 @@ wel::WindowsEventLogHandler ConsumeWindowsEventLog::getEventLogHandler(const std
   LPCWSTR widechar = temp_wstring.c_str();
 
   providers_[name] = wel::WindowsEventLogHandler(EvtOpenPublisherMetadata(NULL, widechar, NULL, 0, 0));
-  logger_->log_trace("Not found the handler -> created handler for %s", name.c_str());
+  logger_->log_info("Not found the handler -> created handler for %s, number of cached handlers: %zu", name.c_str(), providers_.size());
```

Comment:
Let's fix the wording here.
```suggestion
  logger_->log_info("Handler not found for %s, creating. Number of cached handlers: %zu", name, providers_.size());
```